File: selenium-java-framework/src/main/java/pages/CartPage.java
package pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class CartPage {
    WebDriver driver;

    By cartItems = By.className("cart_item");
    By checkoutButton = By.id("checkout");

    public CartPage(WebDriver driver) {
        this.driver = driver;
    }

    public int getCartItemsCount() {
        return driver.findElements(cartItems).size();
    }

    public void clickCheckout() {
        driver.findElement(checkoutButton).click();
    }
}

File: playwright-python-framework/pages/cart_page.py
from utils.base_page import BasePage


class CartPage(BasePage):
    CART_ITEMS = ".cart_item"
    CHECKOUT_BUTTON = "#checkout"

    def get_cart_items_count(self):
        return self.page.locator(self.CART_ITEMS).count()

    def click_checkout(self):
        self.click(self.CHECKOUT_BUTTON)